packages/itwinui-variables/src/sizes.scss

```diff
@@ -12,6 +12,8 @@
   --iui-size-xl: #{math.div(32, 16)}rem;
   --iui-size-2xl: #{math.div(64, 16)}rem;
   --iui-size-3xl: #{math.div(96, 16)}rem;
+
+  --iui-gap-between-inputs: var(--iui-size-s);
```

Comment:
i dont think this needs to go in the variable packages (at least not in this PR). it can just be a sass variable in `tabs.scss`

Reply:
Replaced the variables package variable with a CSS variable in `tabs.scss` prefixed with `_`, similar to [`expandable_block.scss (L6)`](https://github.com/iTwin/iTwinUI/blob/main/packages/itwinui-css/src/expandable-block/expandable-block.scss#L6)

Reply:
it doesn't need to be a css variable. it is used like a constant value, so sass variable will be fine.

in expandable-block we are assigning it a bunch of times (it's "dynamic"), that's why it's a css variable
![](https://user-images.githubusercontent.com/9084735/210570267-fa96f0c5-85e3-4f7a-8911-28e00988b1d6.png)

Reply:
Ohh, right, changed it to a sass variable [`89bdee6` (#860)](https://github.com/iTwin/iTwinUI/pull/860/commits/89bdee6c86699095ef8a8ec0a3575f2db8390976)